docs/releasenotes/release-1.129.md

```diff
@@ -54,4 +54,4 @@ TODO: list contributors with `git log v1.128.0... | grep Merge | grep from | awk
 
 ## Bug Fixes:
 
-* [SAMPLE_Issue 3007](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3007) ComputeBackendService cannot refer clientTLSPolicy due to invalid format (This is a sample, your actual release note should not contain `SAMPLE_`, otherwise it will be deleted)
\ No newline at end of file
+* The `datasetRef` field in the `BigQueryDataTransferConfig` resource should be [optional](https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/3777) rather than required.
```

Comment:
So we have fixed many issues in every release, but we are not aiming at including them all in the release note. Also, I don't think changing this field from required to optional is a bug, it is more of a drift from the rest API, which causes zero regression to Config Connector behavior.  What's more, the 1.129 release note has been published in Google, so it is harder to update  it.

Reply:
Done